.clang-format

```diff
@@ -0,0 +1,203 @@
+---
```

Comment:
I'm leaving this in for now, but it needs to be removed before the PR is ready for review